README.md

```diff
@@ -118,3 +122,115 @@ docs/CONTRIBUTING.md
 
 which contains important information.
 
+
+
+# PT-BR
+## Introdução
+Snaskii é um jogo de cobra ASCII muito simples, uma homenagem ao clássico
+fliperamas dos anos 80. Foi concebido para ser desenvolvido como um exercício de colaboração
+de programação no curso sobre Sistemas de Código Aberto lecionado a
+alunos de graduação em Ciências de Computação.
+
+
+## Começo rápido
+Se você obteve a fonte do projeto do __version control repository__,
+execute o scrpit a seguir:
+
+ ```
+ $ ./bootstrap.sh
+ ```
+
+para impulsionar o script de configuração de compilação `configure`. Para isso, você
+precisa ter GNU Build System (Autotools) instalado. Plataformas baseadas em debian/ubuntu, 
+você pode instalar os softwares necessários com:
+
+```
+$ sudo apt install automake autoconf libtool
+```
+
+Por outro lado, se você obteu o software de __distribution repository__, geralmente como tarball,
+você já deve ter o script `configure`.
+
+De qualquer forma, encontre o arquivo na pasta raíz do diretório fonte e o execute:
+
+```
+ $ ./configure
+```
+
+Este script deve realizar uma série de testes para coletar dados sobre a plataforma build. Se surgirem avisos sobre a falta de alguns softwares, instale-os conforme necessário.
+
+Por exemplo, você precisará de `libncurses`, que no debian/ubuntu pode ser
+instalado com:
+
+```
+sudo apt install libncurses5-dev
+```
+
+O suporte para thread POSIX também é necessário.
+
+Por fim, construa o software e instale-o com:
+
+```
+ $ make
+ $ make install
+```
+
+Isso deve instalar o programa no caminho do sistema. Normalmente os arquivos binários
+serão colocados em `/usr/bin`, e os arquivos de dados em `/usr/share`. Privilégios de administrador (sudo) são necessários para escrever nestes locais.
+
+Opcionalmente, se desejar instalar o software em um local diferente,
+por exemplo, em `/tmp/foo`, execute:
+
+```
+ $ ./configure --prefix=/tmp/foo
+ $ make
+ $ make install
+```
+
+Isso deve instalar o software localmente, neste caso em `/tmp/foo/bin`
+e os arquivos de dados em `/tmp/share`.
+
+Para obter instruções mais detalhadas, consulte o arquivo `INSTALL`
+
+
+## EXECUÇÃO
+```
+ Uso:  snaski [opções]
+
+         Opções
+         
+	 -h              Exibe essa mensagem informativa
+	 -d <caminho>    Seleciona o caminho de dados personalizado
+```
+
+## O Jogo
+ 
+O jogo se passa em uma área retangular onde uma cobra continuamente
+se move em uma das quatro direções: esquerda, direita, para cima e para baixo --- ela nunca
+para de se mover. Conforme a cobra se move ela perde energia e, se toda sua energia se esgotar, a
+cobra morre. Para recuperar energia, a cobra precisa comer pedaços de comida que
+são constantemente substituídos em posições aleatórias.
+
+Tenha cuidado, no entanto. As bordas da arena são eletrificadas e matariam a cobrinha se forem tocadas. Além disso, lembre-se que a cobra é venenosa e também morreria se
```

Comment:
The verbal time is wrong here.
I sugest to use the sentences : "As bordas da arena são eletrificadas e matariam a cobrinha se fossem tocadas" and "Além disso, lembre-se que a cobra é venenosa e também morreria caso se mordesse acidentalmente".